src/main/drivers/barometer/barometer_bmp388.c

```diff
@@ -39,7 +39,12 @@
 
 #if defined(USE_BARO) && (defined(USE_BARO_BMP388) || defined(USE_BARO_SPI_BMP388))
 
+#ifndef BMP388_I2C_ADDR_HW
```

Comment:
This check is repeated in 2 places. It should probably be in a header somehwere else.

Reply:
So it's really strange code, thank you, i fixed it